docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -168,4 +166,4 @@ autonomy deploy from-token [OPTIONS] TOKEN_ID KEYS_FILE
 autonomy deploy from-token --use-goerli 2 keys.json
 ```
 
-Runs the service deployment registered with `TOKEN_ID`=2 in the Görli on-chain protocol. The deployment will be run for as many agents as keys are defined in the `keys.json` file.
+Runs the service deployment registered with `TOKEN_ID`=2 in the Görli on-chain protocol. The deployment will be run for as many agents as keys are defined in the `keys.json` file. To understand how to use various chain profiles refere to `Chain Selection` section on the `autonomy mint` command documentation.
```

Comment:
*refer to

Please, add the # link to chain selection if it exists. (Do not capitalize section titles)